core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/migration/MultiregionalBucketMigration.scala

```diff
@@ -281,16 +281,13 @@ trait MultiregionalBucketMigrationHistory extends DriverComponent with RawSqlQue
       sql"""
         select count(*) from #$tableName m
         join (select id, namespace from WORKSPACE) as w on (w.id = m.#$workspaceIdCol)
-        where m.#$startedCol is not null and m.#$finishedCol is null and exists (
-            /* Only-child migrations are excluded from the max concurrent migrations quota. */
-            select null from WORKSPACE where id <> w.id and namespace = w.namespace
-        )
+        where m.#$startedCol is not null and m.#$finishedCol is null
         """.as[Int].head
 
 // The following query uses raw parameters. In this particular case it's safe to do as the
 // values of the `activeStatuses` are known and controlled by us. In general one should use
 // bind parameters for user input to avoid sql injection attacks.
-    final def nextMigration(onlyChild: Boolean) = OptionT[ReadWriteAction, (Long, UUID, String)] {
+    final def nextMigration(rateLimit: Boolean) = OptionT[ReadWriteAction, (Long, UUID, String)] {
```

Comment:
For PPW, `onlyChild` migrations could run even when the pipeline was rate limited by GCP quotas since `onlyChild` migrations didn't consume any GCP quota. This boolean served two purposes for PPW: (1) ensure that `onlyChild` migrations would still be run when the pipeline was otherwise blocked and (2) to block workspaces that **weren't** `onlyChild` migrations from being selected by `nextMigration`. 

I've repurposed this parameter so that only (2) applies. `rateLimit` will be true if the pipeline is blocked (detected by the presence of certain GCP errors) or if we have reached the limit of concurrent migrations. When `rateLimit` is true, `nextMigration` won't return any workspaces and the pipeline will stop until we are no longer rate limited. 

An alternative I hadn't considered previously would be to check the conditional before calling `nextMigration` and return `None` instead of calling `nextMigration` at all. This alternative feels simpler and clearer to me, so I'm going to make that change